stdlib/_operator.pyi

```diff
@@ -64,8 +64,8 @@ def sub(__a: Any, __b: Any) -> Any: ...
 def truediv(__a: Any, __b: Any) -> Any: ...
 def xor(__a: Any, __b: Any) -> Any: ...
 def concat(__a: Sequence[_T], __b: Sequence[_T]) -> Sequence[_T]: ...
-def contains(__a: Container[Any], __b: Any) -> bool: ...
-def countOf(__a: Iterable[Any], __b: Any) -> int: ...
+def contains(__a: Container[Any], __b: object) -> bool: ...
+def countOf(__a: Iterable[Any], __b: object) -> int: ...
```

Comment:
We can get rid of the other `Any` too, because `Container` and `Iterable` are covariant:

```suggestion
def contains(__a: Container[object], __b: object) -> bool: ...
def countOf(__a: Iterable[object], __b: object) -> int: ...
```